Code/Tools/AssetProcessor/native/AssetDatabase/AssetDatabase.cpp

```diff
@@ -116,7 +116,7 @@ namespace AssetProcessor
             "CREATE TABLE IF NOT EXISTS SourceDependency("
             "    SourceDependencyID            INTEGER PRIMARY KEY AUTOINCREMENT, "
             "    BuilderGuid                   BLOB NOT NULL, "
-            "    Source                        TEXT NOT NULL collate nocase, "
+            "    SourceGuid                    BLOB NOT NULL, "
```

Comment:
Out of curiosity why do we use BLOBs for UUID fields? They don't seem to qualify as LOBs.

Reply:
I'm not sure, that's how its always been and I don't know sqlite well enough to even know what the difference would mean here.  Naively I'd assume its less storage space for the binary form of UUIDs vs text form.